apps/admin/traefik-crds/kustomization.yaml

```diff
@@ -1,21 +1,21 @@
-  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.1.0/traefik/crds/traefik.containo.us_middlewares.yaml
-  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.1.0/traefik/crds/traefik.containo.us_middlewaretcps.yaml
-  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.1.0/traefik/crds/traefik.containo.us_serverstransports.yaml
-  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.1.0/traefik/crds/traefik.containo.us_tlsoptions.yaml
-  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.1.0/traefik/crds/traefik.containo.us_tlsstores.yaml
-  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.1.0/traefik/crds/traefik.containo.us_traefikservices.yaml
\ No newline at end of file
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.io_ingressroutetcps.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.io_ingressroutes.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.io_ingressrouteudps.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.io_middlewares.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.io_middlewaretcps.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.io_serverstransports.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.io_tlsoptions.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.io_tlsstores.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.io_traefikservices.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.containo.us_ingressrouteudps.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.containo.us_ingressroutes.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.containo.us_ingressroutetcps.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.containo.us_middlewares.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.containo.us_middlewaretcps.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.containo.us_serverstransports.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.containo.us_tlsoptions.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.containo.us_tlsstores.yaml
+  - https://raw.githubusercontent.com/traefik/traefik-helm-chart/v23.2.0/traefik/crds/traefik.containo.us_traefikservices.yaml
```

Comment:
Undo this one, this is the base file and you've already got the upgraded version in the file you added

Reply:
Fixed... had forgot to do a ctrl s :(